Neos.FluidAdaptor/Classes/ViewHelpers/Widget/AutocompleteViewHelper.php

```diff
@@ -51,6 +52,7 @@ class AutocompleteViewHelper extends AbstractWidgetViewHelper
      */
     public function render(QueryResultInterface $objects, $for, $searchProperty, array $configuration = ['limit' => 10])
     {
-        return $this->initiateSubRequest();
+        $response = $this->initiateSubRequest();
```

Comment:
This should be just `return $this->initiateSubRequest();` The decision that only the content of the SubResponse is rendered is not responsibility of a concrete ViewHelper.